### PR TITLE
i18n consumer

### PR DESCRIPTION
**Pull request by TikWind**
i18n consumer

### CODE REVIEW

File: src/pages/Consumer/Create.tsx [hunk 1]
```diff
@@ -51,7 +52,7 @@ const Page: React.FC = (props) => {
         .map((item) => item[0]);
       const isValid = Object.keys(plugins).some((name) => authPluginNames.includes(name));
       if (!isValid) {
-        notification.warning({ message: '请启用至少一种身份认证类插件' });
+        notification.warning({ message: formatMessage({ id: 'create.consumer.enable.authentication.plugin' }) });
```
Comment (juzhiyuan) [inline src/pages/Consumer/Create.tsx:55]: The ID starts with create is not a good idea, please use ModuleName.key1.key2 style, e.g ConsumerModule.tip.atLeastOneAuthPlugin

File: src/pages/Consumer/Create.tsx [hunk 2]
```diff
@@ -64,22 +65,22 @@ const Page: React.FC = (props) => {
...
   return (
     <>
-      <PageContainer title={`${(props as any).match.params.id ? '编辑' : '创建'} Consumer`}>
-        <Card bordered={false}>
-          <Steps current={step - 1} style={{ marginBottom: 30 }}>
-            <Steps.Step title="基础信息" />
-            <Steps.Step title="插件配置" />
-            <Steps.Step title="预览" />
+      <PageContainer title={`${(props as any).match.params.id ? formatMessage({ id: 'component.global.update' }) : formatMessage({ id: 'component.global.create' })} Consumer`}>
+        <Card bordered = {false}>
```
Comment (juzhiyuan) [inline src/pages/Consumer/Create.tsx:69]: Extra space

File: src/pages/Consumer/Create.tsx [hunk 3]
```diff
@@ -64,22 +65,22 @@ const Page: React.FC = (props) => {
...
+          <Steps current = {step - 1} style={{ marginBottom: 30 }}>
+            <Steps.Step title = {formatMessage({ id: 'create.consumer.basic.information' })} />
```
Comment (juzhiyuan) [inline src/pages/Consumer/Create.tsx:71]: Same issues here

File: src/pages/Consumer/Create.tsx [hunk 4]
```diff
@@ -64,22 +65,22 @@ const Page: React.FC = (props) => {
...
+            <Steps.Step title = {formatMessage({ id: 'create.consumer.preview' })} />
           </Steps>
 
           {step === 1 && <Step1 form={form1} />}
           {step === 2 && <PluginPage data={plugins} onChange={setPlugins} />}
           {step === 3 && <Preview form1={form1} plugins={plugins} />}
         </Card>
       </PageContainer>
-      <ActionBar step={step} lastStep={3} onChange={onStepChange} />
+      <ActionBar step = {step} lastStep = {3} onChange = {onStepChange} />
```
Comment (juzhiyuan) [inline src/pages/Consumer/Create.tsx:81]: Same issues here

**Comment by TikWind** [comment]
@moonming hello, what is rules  about key name? I follow this:
`'menu.consumer.list': 'List','menu.consumer.create': 'Create Consumer','menu.consumer.edit': 'Edit Consumer'`
image

File: src/pages/Consumer/Create.tsx [hunk 5]
```diff
@@ -15,6 +16,7 @@ const Page: React.FC = (props) => {
   const [step, setStep] = useState(1);
   const [plugins, setPlugins] = useState<PluginPage.PluginData>({});
   const [form1] = Form.useForm();
+  const intl = useIntl();
```
Comment (juzhiyuan) [inline src/pages/Consumer/Create.tsx:19]: const {formatMessage} = useIntl()

File: src/pages/Consumer/locales/zh-CN.ts [hunk 6]
```diff
@@ -3,4 +3,33 @@ export default {
...
+  'consumer.list.edit': '编辑',
+  'consumer.list.delete.confirm': '确定删除该条记录吗？',
+  'consumer.list.confirm': '确定',
+  'consumer.list.cancel': '取消',
+  'consumer.list.delete.success': '删除记录成功',
+  'consumer.list.delete': '删除',
+  'consumer.list.list': 'Consumer 列表',
+  'consumer.list.input': '请输入',
+  'consumer.list.create': '创建',
+};
```
Comment (juzhiyuan) [inline src/pages/Consumer/locales/zh-CN.ts:35]: extra space is needed

File: src/pages/Consumer/Create.tsx [hunk 7]
```diff
@@ -32,7 +34,7 @@ const Page: React.FC = (props) => {
     const { id } = (props as any).match.params;
     (id ? update(id, data) : create(data))
       .then(() => {
-        notification.success({ message: `${id ? '更新' : '创建'} Consumer 成功` });
+        notification.success({ message: `${id ? formatMessage({ id: 'consumer.create.edit' }) : formatMessage({ id: 'consumer.create.create' })} Consumer ${formatMessage({ id: 'consumer.create.success' })}` });
```
Comment (juzhiyuan) [inline src/pages/Consumer/Create.tsx:37]: global.update & global.create are ok.

File: src/pages/Consumer/Create.tsx [hunk 8]
```diff
@@ -51,7 +53,7 @@ const Page: React.FC = (props) => {
         .map((item) => item[0]);
       const isValid = Object.keys(plugins).some((name) => authPluginNames.includes(name));
       if (!isValid) {
-        notification.warning({ message: '请启用至少一种身份认证类插件' });
+        notification.warning({ message: formatMessage({ id: 'consumer.create.enable.authentication.plugin' }) });
```
Comment (juzhiyuan) [inline src/pages/Consumer/Create.tsx:56]: .enableAtLeast1AuthenticationPlugin

**Comment by juzhiyuan** [comment]
@TikWind Could we merge this PR, if ok, please remove the WIP from title. Thanks!

**Comment by TikWind** [comment]
> @TikWind Could we merge this PR, if ok, please remove the WIP from title. Thanks!
I have tested, maybe you can merge this PR, thanks.

**Pull request merged**